src/rpc/xmlrpc.cc

```diff
@@ -36,12 +36,18 @@
 
 #include "config.h"
 
-#ifdef HAVE_XMLRPC_C
+#include <cctype>
+#include <string>
+#include <sstream>
+
+#if defined(HAVE_XMLRPC_C)
 #include <stdlib.h>
 #include <xmlrpc-c/server.h>
+#else
+#include "vendor/tinyxml2/tinyxml2.h"
+#include "utils/base64.h"
 #endif
```

Comment:
Should check if we have tinyxml.

Reply:
Will be handled by the other request to split out into different files